MODULE.bazel

```diff
@@ -4,7 +4,7 @@ module(
     repo_name = "bazel_gazelle",
 )
 
-bazel_dep(name = "bazel_skylib", version = "1.3.0")
```

Comment:
Is this update needed?

Reply:
Not needed. I wasn't sure about warnings regarding minimal version selection with bzlmod.